Formula/magnetix.rb

```diff
@@ -0,0 +1,36 @@
+class Magnetix < Formula
+  desc "Interpreter for Magnetic Scrolls adventures"
+  homepage "http://www.maczentrisch.de/magnetiX/"
+  url "http://www.maczentrisch.de/magnetiX/downloads/magnetiX_src.zip"
```

Comment:
would be good to request a versioned URL since this is eventually going to break